### ISSUE TITLE
a11y - Other items

### ISSUE THREAD

**Issue by dsamojlenko** [cds-snc/c19-benefits-node#225]
This is the remainder of the items from that a11y review, for further discussion.

a)      ~~There might be value in placing visually-hidden H2 headings before the “Français” link, “Back” button and the footer. WET and GCWeb already do it for those kinds of things.~~

b)      HTML sectioning elements are unused in the first and last pages’ main content. I’d recommend wrapping sectioning elements (such as `<section>`) around pairs of headings and their associated content (including subsections) to better convey.

c)       ~~The date in the first page wasn’t coded as a time. I’d recommend modifying it to use a `<time>` element and a datetime="2020-04-09" attribute.~~

d)      The first page’s expandable content contains a link called “an emergency loan<https://travel.gc.ca/assistance/emergency-info/financial-assistance/covid-19-financial-help>”. I’d recommend reducing the linked area to just “emergency loan” since “an” is unrelated to the link’s destination.

~~e)      The icon images’ `<img>` elements use aria-hidden="true" attributes and lack null alt attributes. That’s fine, but isn’t how decorative images are traditionally coded. Coding them in the traditional way (e.g. `<img src="…" alt="">`) might potentially be more interoperable than the status quo.~~

f)        ~~A mix of `<img>` and `<svg>` elements are in use for icon images. The “Back” button and “Start over” link use `<img>`, whereas all the other icons are coded as `<svg>`. Was that intentional? Should things be changed to consistently use one or the other?~~

~~g)      The “Back” buttons is coded in strange way. Its `<button>` element uses an aria-label="Back" attribute. But the inside of the button already contains “Back” text. So the aria-label attribute is unnecessary. I’d recommend removing it.~~

h)      The “Back” button is the first thing in the question pages’ `<main>` element. Is that wise? It might be more appropriate to place it in the header area or move it after the H1 heading (at least in code… might be possible to retain the pre-existing visual layout with flexbox CSS). Just food for thought.

i)        The linked area in the final page’s “Dates of payments are available<https://www.canada.ca/en/revenue-agency/services/child-family-benefits/benefit-payment-dates.html> online.” content is kind of unusual. I’d recommend reducing the linked area to just “Dates of payments” since “are available” is unrelated to the link’s destination.
